packages/kbn-apm-synthtrace/src/cli/utils/synthtrace_worker.ts

```diff
@@ -85,9 +92,10 @@ async function start() {
       apmEsClient,
       logsEsClient,
       infraEsClient,
-      assetsEsClient,
       syntheticsEsClient,
       otelEsClient,
+      entitiesEsClient,
+      entitiesKibanaClient,
```

Comment:
Same here, is kibana client needed for the entities?

Reply:
https://github.com/cauemarcondes/kibana/blob/fe7ccdc7b060b6cb9b277ad32287980eb0b05cfb/packages/kbn-apm-synthtrace/src/scenarios/traces_logs_entities.ts#L44-L45